mock_eq/routes.py

```diff
@@ -1,41 +1,51 @@
-from flask import render_template, redirect, request
+from flask import render_template, request, redirect, flash
 from mock_eq import app
+from pathlib import Path
 
-from sdc.crypto.decrypter import decrypt
-from sdc.crypto.key_store import KeyStore, validate_required_keys
+from mock_eq.common.decrypter import Decrypter
+from mock_eq.common.pubsub import PubSub
 
+import logging
+import structlog
 import json
 
-import logging
+logger = structlog.wrap_logger(logging.getLogger(__name__))
 
-KEY_PURPOSE = "authentication"
-log = logging.getLogger()
-logging.basicConfig()
+_health_check = {}
+if Path("git_info").exists():
```

Comment:
Can you get rid of this git_info stuff?  It's always been fairly pointless in our other repos so no need to duplicate it here